WebApiCustomers/Repository/BaseRepository.cs

```diff
@@ -0,0 +1,98 @@
+﻿using FluentValidation.Results;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WebApiCustomers.Model;
+using WebApiCustomers.Validator;
+
+namespace WebApiCustomers.Repository
+{
+    public class BaseRepository : IBaseRepository
+    {
+        private readonly List<CustomersModel> _customersList = new();
+
+        public virtual List<CustomersModel> GetAll()
+        {
+            return _customersList;
+        }
+
+        public virtual CustomersModel GetById(long id)
```

Comment:
Como esta classe é a implementação do IBaseRepository, as funções dela não devem ter o "virtual"
```suggestion
        public CustomersModel GetById(long id)
```